chore: update docs on adding icons (#250)
twilio-labs/paste: packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx

4. Commit your changes in Abstract and request a review from the Paste design team.

## Adding the icon to the Paste repository

Once the icon is designed and exported, we need to get it in the hands of our developers.
There are two ways for icons to be added into the `@twilio-paste/icons` code package:

1. You can [file an issue on Github](https://github.com/twilio-labs/paste/issues) providing all the
   information you have about the icon. Ideally, this issue would attach the SVG icon generated in
   the above steps. This method is hands-off, but comes at the cost of needing to wait between 2 to 6 weeks
   for us to plan and execute on the work.
2. Alternatively, you can open a Pull Request (PR) on our repository with the required changes.
   This self-service approach would greatly speed up the process of getting your icon into the Paste package
   because it bypasses our team's sprint practices.

### Self-service instructions

1. Fork the Paste repository because PRs can only be open against forks and not branches for security reasons.
2. Clone your newly forked Paste repository:
```
git clone <url>
```
3. Navigate inside the newly cloned repository with `cd paste`

> 4. Move any number of `svg` icon files into the `packages/paste-icons/svg/` folder.

Reviewer: ```suggestion
4. For each new icon you would like to create, add an `svg` icon file into the `packages/paste-icons/svg/` folder.
```

> 5. From the root Paste folder, run this automatic conversion command:

Reviewer: ```suggestion
5. From the root Paste folder, run the following command in your terminal:
```

```
yarn workspace @twilio-paste/icons convert-new
```

> 6. Verify the `packages/twilio-icons/src` folder to see if your new icons were added correctly.

Reviewer: ```suggestion
6. Verify your new icons were added correctly by checking the `packages/twilio-icons/src` folder to see the newly  created source files.
```

7. At this point, run the following command to update our documentation with the new icons:
```
yarn workspace @twilio-paste/icons list-icons
```

> 8. Commit your changes and submit a new PR on Github!

Reviewer 2: On the actual page, the numbering resets after every code block.
